packages/suite-data/src/browser-detection/index.ts

```diff
@@ -140,7 +124,7 @@ window.addEventListener('load', () => {
     const updateFirefox = getMainHtml({
```

Comment:
Shouldn't Firefox be removed from `supportedBrowsers` and this function deleted?

Reply:
We're not making Firefox unsupported with this change yet. We're just not recommending it anymore